app/controllers/EnumeratorExamples.scala

```diff
@@ -1,31 +0,0 @@
-package controllers
-
-import play.api.mvc.Controller
-import play.api.mvc.Action
-import play.api.libs.iteratee.Enumerator
-import play.api.libs.concurrent.Promise
-import play.api.libs.concurrent.Execution.Implicits._
-
-object EnumeratorExamples extends Controller {
```

Comment:
Sorry, I was just deleting these files so I can focus on the concept itself. Brilliant video btw!